### PR TITLE
Complete onboarding

### PR DESCRIPTION
**Pull request by kiroj305**
This pr will basically finalize the onboarding screens implementation of Easel.

### CODE REVIEW

**Review by mustafapylons — APPROVED** [review]
Looks good to me.

File: lib/screens/welcome_screen.dart [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                color: EaselAppTheme.kDarkText,
+                fontSize: 16.sp,
+                fontWeight: FontWeight.w400),
+          ),
+          SizedBox(height: 60.h),
+          Container(
+            alignment: Alignment.topLeft,
+            child: PylonsButton(
+              onPressed: () {
+                Navigator.push(
```
Comment (kjawadDeveloper2) [inline lib/screens/welcome_screen.dart]: Kindly make name routes.

File: lib/screens/tutorial_screen.dart [hunk 2]
```diff
@@ -166,7 +166,7 @@ class _TutorialScreenState extends State<TutorialScreen> {
                       Navigator.push(
```
Comment (kjawadDeveloper2) [inline lib/screens/tutorial_screen.dart]: Kindly make name routes.

**Review by kjawadDeveloper2 — CHANGES_REQUESTED** [review]
Kindly solve this issue.

**Comment by kiroj305** [comment]
@kjawadDeveloper2 , migrated to the named routes navigation as per your request. Will be gradually making all other navigations use named routes.

**Comment by kjawadDeveloper2** [comment]
@Tiny-Angel305  did you pushed the code?

**Comment by kiroj305** [comment]
@kjawadDeveloper2 , sorry I just pushed the fixes.

**Review by kjawadDeveloper2 — APPROVED** [review]
Looking good from my side.

**Pull request merged**